resources/views/layout.blade.php

```diff
@@ -53,20 +53,24 @@
             <div class="col">
                 @auth
                     <nav class="d-flex justify-content-end">
-                        <div class="nav-item me-3 mt-2">
-                            @if(Role::find(Auth::user()->role_id)->role_name === "commercial")
-                                <a href="{{route('commercial.contract')}}" class="text-uppercase" style="text-decoration: none">Contract</a>
-                            @endif
-                        </div>
-                        <div class="nav-item me-3 mt-1">
-                            <form action="{{ route('users.edit', Auth::user()->id) }}" method="GET">
-                                <button type="submit" class="btn btn-primary">Profiel</button>
-                            </form>
-                        </div>
-                        <div class="nav-item ms-3 mt-1">
-                            <div class="me-lg-5">
-                                <span class="text-uppercase" style="font-size: 1.5em">{{ Auth::user()->name }}</span>
-                            </div>
+                        <div class="nav-item ms-3 mt-1 dropdown">
+                                <button class="btn dropdown-toggle text-uppercase" style="font-size: 20px" type="button" id="dropdownMenuButton1" data-bs-toggle="dropdown" aria-expanded="false">
+                                    {{ Auth::user()->name }}
+                                </button>
+                            <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton1">
+                                @if(Role::find(Auth::user()->role_id)->role_name === "commercial")
+                                    <li><a href="{{route('commercial.contract')}}" class="text-uppercase dropdown-item" style="text-decoration: none">Contract</a></li>
+                                @endif
+                                @if(Role::find(Auth::user()->role_id)->role_name === "customer")
+                                        <li><a href="{{route('customer.favorites')}}"
+                                               class="text-uppercase dropdown-item" style="text-decoration: none">Favorieten</a>
+                                        </li>
+                                        <li><a href="{{route('customer.purchases')}}"
+                                               class="text-uppercase dropdown-item" style="text-decoration: none">Bestellingen</a>
+                                        </li>
+                                @endif
```

Comment:
Iedereen kan bestellingen doen.

Het enige verschil tussen de 3 types gebruiker is dat customer niet kan adverteren. en zakelijk dat contract nodig heeft.
Nogmaals. we moetne niet zo veel verschil gaan maken tussen de types klanten.